src/content-scripts/gemini-tracker.js

```
@@ -302,15 +302,197 @@
+            // If logging is globally disabled, return false
+            if (!config.enabled) {
+                return false;
+            }
+            
+            // If the log level is disabled, return false
+            if (level && !config.levels[level]) {
+                return false;
+            }
+            
+            // If the component is explicitly configured, use that setting
+            if (component && config.components.hasOwnProperty(component)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)